File: 백준/Silver/14425. 문자열 집합/문자열 집합.py
first=list(map(int,input().split()))
set1=set()
set2=set()
set3=set()
count=0
for i in range(first[0]):
    word=input()
    set1.add(word)
for i in range(first[1]):
    if input() in set1:
        count+=1
print(count)

File: 백준/Silver/14425. 문자열 집합/문자열 집합.java
import java.util.HashSet;
import java.util.Scanner;
import java.util.Set;
public class Main {
    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);
        int count=0;
        int first = sc.nextInt();
        int second = sc.nextInt();
        Set<String>set1 = new HashSet<>();
        for(int i=0;i<first;i++){
            String a = sc.next();
            set1.add(a);
        }
        for(int i =0;i<second;i++) {
            String a = sc.next();
            if (set1.contains(a)) count++;
        }
        System.out.println(count);
    }
}